### PR TITLE
Added withdraw ether for owner. This will help to close the issue number #14

### PR DESCRIPTION
**Pull request by Saurabhpatil-dev**
I have added the withdraw ether method which will withdraw ether only when the contract has ether and the wallet connected is of Owner.

### CODE REVIEW

**Comment by Saurabhpatil-dev** [comment]
This PR is regarding issue #14 .

Please review an let me know if any changes are required.

File: Decentralized-Autonomous-Organizations.md [hunk 1]
```diff
@@ -875,6 +890,9 @@ export default function Home() {
         getDAOTreasuryBalance();
         getUserNFTBalance();
         getNumProposalsInDAO();
+        if (parseFloat(treasuryBalance) > 0) {
```
Comment (haardikk21) [inline Decentralized-Autonomous-Organizations.md:893]: This should not be here. Why would you withdraw ether everytime balance is > 0. This should be on a button click or something

**Review by haardikk21 — CHANGES_REQUESTED** [review]

File: Decentralized-Autonomous-Organizations.md (hunk 1 above)
Comment (Saurabhpatil-dev) [reply Decentralized-Autonomous-Organizations.md:893]: Yes, you are right. Let me make some change like if Owner and contract has some eth then we render the withdraw button, else we don't render it. That would be fine ?

Comment (Saurabhpatil-dev) [reply Decentralized-Autonomous-Organizations.md]: Hi @haardikk21 , 

I have made the changes and committed them. Can you please review and test them from your side ?

Please let me know what do you think.

Some Screen Shots for reference:

View for Owner :
-----------------
Screenshot 2022-09-30 at 2 00 30 PM

View for owner after withdrawn:
-------------------------------

Screenshot 2022-09-30 at 2 02 56 PM


View for no-owner: 
-------------------
Screenshot 2022-09-30 at 1 57 09 PM

**Comment by yanukadeneth99** [comment]
Hey @Saurabhpatil-dev, I think this addition will be great. I'll try running the code and checking.

File: Decentralized-Autonomous-Organizations.md [hunk 2]
```diff
@@ -819,6 +822,37 @@ export default function Home() {
...
+  // Calls the `withdrawEther` function in the contract
+  const withdrawEther = async () => {
+    try {
+      const signer = await getProviderOrSigner(true);
+      const daoContract = getDaoContractInstance(signer);
+      const tx = await daoContract.withdrawEther();
+      setLoading(true);
+      await tx.wait();
+      setLoading(false);
+      await getOwner();
```
Comment (yanukadeneth99) [inline Decentralized-Autonomous-Organizations.md:851]: I don't see a reason why you'd want to run `getOwner()` again here. Instead, maybe you'd want to run the `getDAOTreasuryBalance()` to update the Balance in the contract

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**